client/my-sites/upgrades/domain-management/email-forwarding/email-forwarding-item.jsx

```diff
@@ -29,9 +29,41 @@ const EmailForwardingItem = React.createClass( {
 			this.recordEvent( 'deleteClick', domain, mailbox, forward_address, ! Boolean( error ) );
 
 			if ( error ) {
-				notices.error( error.message );
+				notices.error( error.message || this.translate( 'Failed to delete email forwarding record. Please try again or contact customer support if error persists.' ) );
 			} else {
-				this.props.successNotice( this.translate( 'Yay, %(email)s has been successfully deleted!', { args: { email: email } } ) );
+				notices.success(
+					this.translate( 'Yay, %(email)s has been successfully deleted!', {
+						args: {
+							email: email
+						}
+					} ), {
+						duration: 5000
+					} );
+			}
+		} );
+	},
+
+	resendVerificationEmail: function() {
+		const { temporary, domain, mailbox, forward_address, email } = this.props.emailData;
+
+		if ( temporary ) {
+			return;
+		}
+
+		upgradesActions.resendVerificationEmailForwarding( domain, mailbox, ( error, response ) => {
+			this.recordEvent( 'resendVerificationClick', domain, mailbox, forward_address, ! Boolean( error ) );
+
+			if ( error || ! response.sent ) {
+				notices.error( error.message || this.translate( 'Failed to resend verification email for email forwarding record. Please try again or contact customer support if error persists.' ) );
+			} else {
+				notices.success(
+					this.translate( 'Yay, successfully sent confirmation email to %(email)s!', {
```

Comment:
Maybe, not yay at this point and clarify they need to go an confirm before forwarding starts.